radix-engine-tests/tests/application/stake_reconciliation.rs

```diff
@@ -19,7 +19,10 @@ fn test_stake_reconciliation() {
     // Arrange
     let pub_key = Secp256k1PrivateKey::from_u64(1u64).unwrap().public_key();
     let mut ledger = LedgerSimulatorBuilder::new()
-        .with_protocol_version(ProtocolVersion::Babylon)
+        .with_custom_protocol(|builder| builder
+            .with_babylon(BabylonSettings::test_minimal())
+            .only_babylon()
```

Comment:
`with_babylon` + `only_babylon` is confusing to me. Based on what I can tell, this is giving configuration to the babylon protocol update and then specifying to only go up to babylon protocol update. Is that right?

Reply:
That’s correct!

But you’re right, it is confusing! Perhaps calling the settings method configure_babylon would be better?